knimin/templates/pm_plate_list.html

```diff
@@ -133,7 +129,7 @@ <h3 id="page-title">{{target[0].upper() + target[1:]}} Plate List</h3>
           <i class="fa fa-battery-empty"></i>
           {% end %}
         </span></td>
-      <td title="{{plate['study'][3]}}">{{plate['study'][1]}}</td>
+      <td title="{{plate['studies'][0]}}">{{plate['studies'][0]}}</td>
```

Comment:
Can you add a comment about the magic numbers? -- From the current context is not clear why [3] -> [0] and [1] to [0]